<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KisCustomConvolutionFilterConfigurationBaseWidget</class>
  <widget class="QWidget" name="KisCustomConvolutionFilterConfigurationBaseWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>138</width>
        <height>230</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Custom Convolution Filter Configuration Widget</string>
    </property>
    <layout class="QHBoxLayout" >
      <item>
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="KisMatrixWidget" name="matrixWidget" />
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="QLabel" name="textLabelFactor" >
                  <property name="text" >
                    <string>Factor:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QSpinBox" name="spinBoxFactor" >
                  <property name="minimum" >
                    <number>1</number>
                  </property>
                  <property name="value" >
                    <number>1</number>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer1" >
                  <property name="sizeHint" >
                    <size>
                      <width>21</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="QLabel" name="textLabelOffset" >
                  <property name="text" >
                    <string>Offset:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QSpinBox" name="spinBoxOffset" >
                  <property name="maximum" >
                    <number>255</number>
                  </property>
                  <property name="minimum" >
                    <number>-255</number>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer2" >
                  <property name="sizeHint" >
                    <size>
                      <width>24</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="spacer24" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>40</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="spacer5" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction></pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>KisMatrixWidget</class>
      <extends></extends>
      <header location="local" >kis_matrix_widget.h</header>
      <sizehint>
        <width>150</width>
        <height>100</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0" >
      <data format="PNG" length="1125" >89504e470d0a1a0a0000000d4948445200000016000000160806000000c4b46c3b0000042c49444154789cb5954f6c14551cc73fefcd7476b65bdaae4bb78bb5502a14d404e4801c88182d1c4c2c693da847400f9c24c68b878684238660e2b1e01f12c19493012ef2478c814412d354a46017a8a564bb6da5bbedccee767776e63d0ffb073751d483bfe49799974c3eeffb7ebf37df9fd05a530b2184040cc0042420aaf9a4d0d554800f045a6b256ae0e1e1e1d6bebebe838ee31c48a7d39b5cd7fd075e251cc7617272f2ded8d8d819cff33e0316819259537aead4a9839d5dd6d1784f91f55b0a94830242088404d304292bef68a89f520802a598fecddaa04f1a876f5c250c7c0a64cdeac686e33807e23d45e6b297c8b877f1831542614550b6599835c83c2a81b6786a75134faf2f1169f12997350881d9021d0903e06de0745d3160a6d3e94dbd5b0a64dcbb94b5831d0e3375ab892b1772dcf9790528543f8dd0d367b36768153b5e31503a0f1aecb004580b44ffac58baae8b1714f0833c7638cc8dab303a320f4822ab4c7a37c69196203de3319d5ce1c4d13c733331dedc67a129a154fd128401ab0616d55a130ac3d42d93d1913940d13fd0c9ee0183685c60da01c5421bd72f7a8c8efccef9afd374267ad93d642365be0636a0d28ec7600941d9e6f23917f0e97f23ce5bef35d19ec863da0ed9059b2be70bec196c66dfa10ec0e49b338f7017258651bf95021035c595429bb0903248fe52a2b5b595dd7b4d945cc2340cdca536be389ee3f67886c5798f773fe8e0dac508c989659277a2180da4ca4ff07821058b8b251445d63d6b13ed1098a6417e39cac85197dbe31962ab9bd9f1f22a226d45366f6d0620fdb08c900d281af6110284b20085b414861d905d88f2e52739ee8cbb8022143259d3dd84691730aa2d52da441a8de0c6958068870022a41e9629ad3473fd3b8fdbe319dadb9b4924da994d2d716c7896fbe35152f78b48245d6b2da4507faf582be8eaf159b721cc837b05ae7debb1f79d08cb8b515edad942a22bc4b1c33eb3d34b1c797f06af90a72d16e2f96d9a74aa11dca8586b222d01af0fb60070f6c402d72f15d97f28c6f6d7027a5f5ce6c3233dc4e2ede496b278be4fff608cee8d3e1add806aeca51094cbb06397c1ecc328e746537c7e3ccdb5cb1136bf60635882d4d41c6ec6836ab37efa214f72208ed9f4d7cdd38ee310280542e38b1c43fb6de26b3672e1ec3cc99bcb246f66a938a3241ab3e91f7c861fbf77710b1e5e49915bae974203ba0e9e9c9cbc373d6d6d305a040a89c2a77f50b27d5782bbbf7acccf28349235dd16cf6dd374f7295e1de8a45c02d37499182b01cc0201a085d61a2144d8b2ac8fb6ed340e77240c4261890e04c250185262546d534a032154b59e0ad394e41c98182bf268ce6721ed9f064e0253356f6da2e24c1f030f783c15fe6da680af8021602bd051532ca9b8521488559f61aa86c29343578fbf0264a94c906c7d3409214c20043457a116ff6de6795578012889ff6b98fe016ea0ce1c203e47720000000049454e44ae426082</data>
    </image>
  </images>
</ui>
